mllam_data_prep/create_dataset.py

```diff
@@ -184,6 +218,18 @@ def create_dataset(config: Config):
 
         dataarrays_by_target[target_output_var].append(da_target)
 
+    # validate projections across input datasets
+    if projections:
+        try:
+            validate_projection_consistency(projections)
+        except ProjectionInconsistencyWarning as e:
+            logger.warning(f"Projection information might be ambiguous: {e}")
+        projection = pyproj.CRS.from_cf(projections[0])
+
+        # TODO: generalize the retrieval of x and y coords
+        # coords = (dataarrays_by_target[target_output_var][0]['x'], dataarrays_by_target[target_output_var][0]['y'])
+        # lon, lat = get_latitude_longitude_from_projection(projection, coords)
```

Comment:
@leifdenby this todo was just an example on where the projection information is available to derive lat and lon. I can delete this for now as this is not yet supported with mllam-data-prep